notebooks/en/llm_judge.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>Perhaps it would be useful to give some guidance on using LLM-as-a-judge in production - some basic dos and don'ts?. E.g. Should it be used in replacement of human evaluators, or as assistance for them? How should one monitor LLM-as-a-judge? (If the questions drift a lot, will that affect the system). And so on.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>